docs/core/introduction.md

```diff
@@ -179,231 +72,175 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
 
-The .NET JIT compiler can do *tiered compilation*, which means it can recompile individual methods at run time. This feature lets it compile quickly while still being able to produce a highly tuned version of the code for frequently used methods.
+The .NET Team at Microsoft works collaboratively with other organizations to distribute and support .NET in various ways.
 
-For more information, see [Managed execution process](../standard/managed-execution-process.md) and [Tiered compilation](whats-new/dotnet-core-3-0.md#tiered-compilation).
+[Red Hat supports .NET](https://developers.redhat.com/topics/dotnet/) on Red Hat Enterprise Linux (RHEL).
 
-### AOT compiler
+[Samsung supports .NET](https://developer.tizen.org/development/training/.net-application) on Tizen platforms.
 
-The default experience for most .NET workloads is the JIT compiler, but .NET offers two forms of ahead-of-time (AOT) compilation:
+## Runtime
 
-* Some scenarios require 100% AOT compilation. An example is [iOS](/xamarin/ios/).
-* In other scenarios, most of an app's code is AOT-compiled but some is JIT-compiled. Some code patterns aren't friendly to AOT (like generics). An example of this form of AOT compilation is the [ready-to-run](whats-new/dotnet-core-3-0.md#readytorun-images) publish option. This form of AOT offers the benefits of AOT without its drawbacks.
+The [Common Language Runtime (CLR)](../standard/clr.md) is the foundation all .NET apps are built on. The [fundamental features of the runtime](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/intro-to-clr.md) are:
 
-### Automatic memory management
+* Garbage collection.
+* Memory safety and type safety.
+* High level support for programming languages.
+* Cross-platform design.
 
-The *garbage collector* (GC) manages the allocation and release of memory for applications. Each time your code creates a new object, the CLR allocates memory for the object from the [managed heap](../standard/garbage-collection/fundamentals.md#the-managed-heap). As long as address space is available in the managed heap, the runtime continues to allocate space for new objects. When not enough free address space remains, the GC checks for objects in the managed heap that are no longer being used by the application. It then reclaims that memory.
+.NET is sometimes called a "managed code" runtime. It is called *managed* primarily because it uses a garbage collector for memory management and because it enforces type and memory safety. The CLR is also virtualizes (or abstracts) a variety of operating system and hardware concepts, such as memory, threads, and exceptions.
```

Comment:
```suggestion
.NET is sometimes called a "managed code" runtime. It is called *managed* primarily because it uses a garbage collector for memory management and because it enforces type and memory safety. The CLR virtualizes (or abstracts) a variety of operating system and hardware concepts, such as memory, threads, and exceptions.
```